cmds/bginfo/main.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017-2018 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// fspinfo prints FSP header information.
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/linuxboot/fiano/pkg/intel/metadata/bg/bgbootpolicy"
+	"github.com/linuxboot/fiano/pkg/log"
+)
+
+var (
+	flagJSON = flag.Bool("j", false, "Output as JSON")
+)
+
+func main() {
+	flag.Parse()
+	if flag.Arg(0) == "" {
+		log.Fatalf("missing file name")
+	}
+	data, err := os.ReadFile(flag.Arg(0))
+	if err != nil {
+		log.Fatalf("cannot read input file: %x", err)
+	}
+
+	ACBPMagic := []byte("__ACBP__")
```

Comment:
nit: acbpMagic is more proper name https://go.dev/wiki/CodeReviewComments#initialisms there are no point to export variable from main